test/fixtures/Events.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "operation": "check in",
+    "timestamp": "Fri Feb 26 2016 09:24:20 GMT+0000 (GMT)",
+    "detainee": {
```

Comment:
pretty sure you can reference the detainee ID in the fixtures so long as the sharedbootstrap has the models in the right order. so you can do `"detainee":"bigone_345"` and it'll magically link to the detainee with that ID rather than having to redeclare. we do this for movements + centers if you want a reference